src/stores/AppStore.js

```diff
@@ -758,6 +758,20 @@ export default types
       await getEnv(self).events.invoke('nextTask');
     }
 
+    // nextTask will only call the next task in taskHistroy.  We need to
+    // invoke the next task in the whole task list.
+    function nextTaskInList() {
+      // TODO: we should find better way to access the dataManager's Mobx Store
+      const taskList = window.dataManager.store.taskStore.list;
+      const thisTaskIdx = taskList.findIndex((x) => x.id === self.task.id);
```

Comment:
Is this the task list for the specific tab we came from? Or all of them?

Reply:
Good question!  Conveniently this is the list showed in the datamanager table, which is what is in the tab 🎉 

I will add a comment on this!